components/Footer.vue

```diff
@@ -1,62 +1,55 @@
+            </nav>
+        </div>
+        <div class="middle-footer-section flex flex-1 align-middle justify-center">
+            <span class="self-center">{{ $t('footer.poweredBy') }}</span>
+            <nav class="flex mx-2 self-center">
+                <NuxtLink to="https://www.netlify.com/" target="_blank"
+                    class="underline hover:text-primary-hover transition-colors" data-testid="netlify-link">Netlify
+                </NuxtLink>
+            </nav>
+        </div>
+        <div class="right-footer-section flex flex-0 flex-row gap-6 justify-end px-8 text-xs">
+            <div class="mx-2 hover:text-primary-hover transition-colors">
+                <NuxtLink to="https://github.com/ourjapanlife/findadoc-web/" target="_blank" class="flex flex-col flex-wrap"
+                    data-testid="github-link">
+                    <div>{{ $t('footer.contribute') }}</div>
+                    <div class="self-center">GitHub</div>
+                </NuxtLink>
+            </div>
+            <span class="self-center">·</span>
+            <div class="flex flex-col flex-wrap hover:text-primary-hover transition-colors">
+                <span>{{ $t('footer.feedback') }}</span>
+                <NuxtLink to="https://forms.gle/4E763qfaq46kEsn99" target="_blank" class="px-2 mx-auto underline font-bold"
+                    data-testid="feedback-link">{{ $t('footer.clickHere') }}</NuxtLink>
+            </div>
+        </div>
     </div>
-    <div class="middle-footer section flex flex-1 align-middle justify-center">
-      <span class="self-center">{{$t('footer.poweredBy')}}</span>
-      <nav class="flex mx-2 self-center">
-        <NuxtLink
-          to="https://www.netlify.com/"
-          target="_blank"
-          class="underline hover:text-primary-hover transition-colors"
-          data-testid="netlify-link"
-        >Netlify</NuxtLink>
-      </nav>
-    </div>
-    <div class="right-footer-section flex flex-0 flex-row gap-6 justify-end px-8 text-xs">
-      <div class="mx-2 hover:text-primary-hover transition-colors">
-        <NuxtLink
-          to="https://github.com/ourjapanlife/findadoc-web/"
-          target="_blank"
-          class="flex flex-col flex-wrap"
-          data-testid="github-link"
-        >
-          <div>{{$t('footer.contribute')}}</div>
-          <div class="self-center">GitHub</div>
-        </NuxtLink>
-      </div>
-      <span class="self-center">·</span>
-      <div class="flex flex-col flex-wrap hover:text-primary-hover transition-colors">
-        <span>{{$t('footer.feedback')}}</span>
-        <NuxtLink
-          to="https://forms.gle/4E763qfaq46kEsn99"
-          target="_blank"
-          class="px-2 mx-auto underline font-bold"
-          data-testid="feedback-link"
-        >{{$t('footer.clickHere')}}</NuxtLink>
-      </div>
-    </div>
-  </div>
 </template>
+<style>
+@media screen and (max-width: 640px) {
```

Comment:
we should be able to implement these in tailwind style instead of using the style section. check out how then do media queries.